### ISSUE TITLE
Questions on Ora_Migrator

### ISSUE THREAD

**Issue by naveenjul29** [cybertec-postgresql/ora_migrator#25]
Will Ora_migrator support migrating partitioning and sub-partioning of tables, index and LOBS?
Will Ora_migrator support migrating package, package body and materialized views?

What is the major differences between Ora2Pg and Ora_Migrator?

**Comment by laurenz** [comment]
There is no direct support for migrating partitioning, but you can use the foreign table created by `db_migrate_mkforeign`, create a partitioned table in PostgreSQL yourself and use `INSERT INTO ... SELECT ...` to migrate the data.

LOBs are supported and should work fine (they will be migrated to `text` and `bytea`, respectively), but migration of big tables with LOBs in them is not very performant, because due to an Oracle limitation a round trip to the Oracle server is required for each migrated row.

There is no support whatsoever for migrating PL/SQL code, you'll have to translate the code yourself.

Basically, ora_migrator does those 90% that are simple, all the complicated stuff requires extra processing.

The major difference to ora2pg is that oracle_fdw uses foreign tables to extract the PostgreSQL metadata and migrate the data.
Also, which ora2pg is a client tool that accesses both databases, ora_migrator runs inside the target database.

**Comment by naveenjul29** [comment]
Thank you so much for quick response.
